src/ab-test-preview.js

```diff
@@ -0,0 +1,44 @@
+import { getLetter } from '../src/utils';
+
+const queryString = window.location.search;
+const urlParams = new URLSearchParams(queryString);
+const abtest = urlParams.get( 'set_test' );
+const abIndex = parseFloat( urlParams.get( 'i' ) );
```

Comment:
What was the reason for choosing `parseFloat()` over `parseInt()` here?

Reply:
In my latest commit I have changed this approach with the existing preview link structure. Only really fighting an issue to extract the relevant info from the URL. I'll comment on the section I have an issue with.

Reply:
This was updated an changed to a different method